draft-yates-threat-exposure-notification-protocol-00.xml

```diff
@@ -1582,21 +1581,20 @@ Content-Type: application/json
 Content-Language: en
 
 {
-    "keys": {
-        "https://tenprotocol.org/keys/btle": [
-            "1478E46F897749C691A28351F0B054F1",
-            "20A7412C65A046DCB5538D7DA45A6B5D",
-            "24BFE7A43563495BA36C79A96617A602",
-            "371108A73B964915A62D81E49A783F85",
-            "4075DB637A914896B5459EF3E377AF0D",
-            "5F51F91099B84C9685CD268941A3DEEC",
-            "67684C56F9A6407A99DB57E6B871EC1D",
-            "AA0817D30C64407CB4F5DE1425336922",
-            "CFC046DEF50244D58CA58A0AD7FED287",
-            "D555096A727C49FEADD03488154E6152",
-            "E48A37FB87924885AD6879857DC297B2"
-        ]
-    }
+    "key_type" : "https://tenprotocol.org/keys/btle",
+    "keys": [
+        "1478E46F897749C691A28351F0B054F1",
+        "20A7412C65A046DCB5538D7DA45A6B5D",
+        "24BFE7A43563495BA36C79A96617A602",
+        "371108A73B964915A62D81E49A783F85",
+        "4075DB637A914896B5459EF3E377AF0D",
+        "5F51F91099B84C9685CD268941A3DEEC",
+        "67684C56F9A6407A99DB57E6B871EC1D",
+        "AA0817D30C64407CB4F5DE1425336922",
+        "CFC046DEF50244D58CA58A0AD7FED287",
+        "D555096A727C49FEADD03488154E6152",
+        "E48A37FB87924885AD6879857DC297B2"
+    ]
```

Comment:
We might want to think of adding before/after to allow the implementation to page results and allow for cursoring.

Reply:
Good call